modules/openxBidAdapter.js

```diff
@@ -1,18 +1,30 @@
 const bidfactory = require('src/bidfactory.js');
 const bidmanager = require('src/bidmanager.js');
 const adloader = require('src/adloader');
+const ajax = require('src/ajax');
 const CONSTANTS = require('src/constants.json');
 const utils = require('src/utils.js');
 const adaptermanager = require('src/adaptermanager');
 
 const OpenxAdapter = function OpenxAdapter() {
   const BIDDER_CODE = 'openx';
   const BIDDER_CONFIG = 'hb_pb';
+  const BIDDER_VERSION = '1.0.1';
   let startTime;
+  let timeout = $$PREBID_GLOBAL$$.bidderTimeout;
```

Comment:
We are trying capture the timeout at the time of request and store it until time of response. Do you know if this global stays constant throughout? We have observed that sometimes it seems the value changes so that's why I'm doing this to make sure.